TestMain.py

```diff
@@ -10,8 +10,12 @@ class TestMain(unittest.TestCase):
     def setUpClass(cls):
         # cloning the specimin 
         main.clone_repository('https://github.com/kelloggm/specimin.git', 'resources')
-        cls.json_data = main.read_json_from_file('resources/test_data.json')[0]
-        cls.specimin_dir = "resources/specimin"
+        cls.json_data = main.read_json_from_file('resources/test_data.json')[6]
```

Comment:
Why does this index need to change from 0 to 6? This seems unrelated to the rest of the PR to me (but I'm not a python expert), and it's unexplained.

Reply:
Specimin is not successful with the first issue (cf-1291) of the issue pool. that is why selecting a target that specimin passes.

cf-1291 fails when I use the absolute path for the targets root directory. Which is really weird. Tough the error make sense. But why it is successful while relative path is  used and fails for absolute path I don't understand that

Reply:
I looked into this a bit, and discovered that the failure should have been expected: cf-1291 has a wildcard import (in the class that the target class extends). The more interesting question is why it ever worked at all: it should not have.

Reply:
(In particular, the file `../specimin-evaluation/ISSUES/cf-1291/input/daikon/java/daikon/chicory/DaikonVariableInfo.java` on my copy has a wildcard import statement.)

Reply:
Yes Professor, there are wildcard import statements in the parent class of the target. I will investigate that in a separate issue. Added a TODO there